docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -83,17 +83,17 @@ This tutorial shows only one possible use case. There are many more use cases on
    ```
    >**NOTE:** The `sap.kyma.custom.acme.payload.sanitised.v1` is a sample event type declared by the emitter Function when publishing events. You can choose a different one that better suits your use case. Keep in mind the constraints described on the [Event names](../../05-technical-reference/evnt-01-event-names.md) page. The receiver subscribes to the event type to consume the events.
 
-   >**NOTE:** The event object provides convenience functions to build and publish events. To send the event, build the Cloud Event. To learn more, read [Function's specification](../../05-technical-reference/svls-08-function-specification.md#event-object-sdk). In addition, your **eventOut.source** key must point to `“kyma”` to use Kyma in-cluster Eventing.
+   >**NOTE:** The event object provides convenience functions to build and publish events. To send the event, build the Cloud Event. To learn more, read [Function's specification](../../05-technical-reference/svls-07-function-specification.md#event-object-sdk). In addition, your **eventOut.source** key must point to `“kyma”` to use Kyma in-cluster Eventing.
    >**NOTE:** There is a `require('axios')` line even though the Function code is not using it directly. This is needed for the auto-instrumentation to properly handle the outgoing requests sent using the `publishCloudEvent` method (which uses `axios` library under the hood). Without the `axios` import the Function still works, but the published events are not reflected in the trace backend.
 
-4. Apply your emitter Function:
+1. Apply your emitter Function:
 
     ```bash
     kyma apply function
     ```
    Your Function is now built and deployed in Kyma runtime. Kyma exposes it through the APIRule. The incoming payloads are processed by your emitter Function. It then sends the sanitized content to the workload that subscribes to the selected event type. In our case, it's the receiver Function.
 
-5. Test the first Function. Send the payload and see if your HTTP traffic is accepted:
+2. Test the first Function. Send the payload and see if your HTTP traffic is accepted:
```

Comment:
Why change points 4 and 5 to 1 and 2? Don't they follow steps 1-3 in the "Create the emitter Function" section?

Reply:
Oh, I didn't want to change those. It must have been an automatic change in VSC that I didn't notice. I will revert it